README.md

```diff
@@ -157,4 +156,8 @@ are as follows:
 is a compiled language so there is no standard linter.
 * Terraform - terraform has a built-in linter in the 'terraform validate'
 command.
-* Dockerfiles - hadolint. Can be found in homebrew
\ No newline at end of file
+* Dockerfiles - hadolint. Can be found in homebrew
+
+## Compatibility
```

Comment:
This section should be moved above the Usage section. Also, the URL for the 1.0.0 link to the registry must be included.

Reply:
Fixed